lib/Scheduler/sched_region.cpp

```diff
@@ -320,6 +325,9 @@ FUNC_RESULT SchedRegion::FindOptimalSchedule(
   else
     CmputLwrBounds_(false);
 
+  // Set the lowerbound of spill cost (need to CmptCostLWrBound() first)
+  SpillCostLwrBound_ = getSpillCostLwrBound();
```

Comment:
This might be redundant. It is setting `SpillCostLwrBound_ = SpillCostLwrBound_`

Reply:
Yeah I think this is redundant. The reason why, however, is that CmputLwrBounds_ is coded weirdly. It has the side effect of setting the spill LB yet it returns the cost LB. Would it be better if we set both, return neither, and rename the function cmputeAndSetLB. This would help avoid the redundancy error in the future.

Reply:
I am fine with renaming it `cmputeAndSetLB` since that would make more sense.